UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs

```diff
@@ -14,8 +14,7 @@
 public class ReactionManager : MonoBehaviour
 {
 	public float n = 20;
-
-	[SerializeField]
+	
 	private GameObject fireLight = null;
```

Comment:
this one is not setteable in inspector anymore?

Reply:
Doesnt need to be, its taken from the atmos manager instead

Reply:
cool, are you gonna make any more changes? If not, I'm merging.

Reply:
Cus every matrix has its own reaction manager, while theres only one atmos manager. I just cache it there instead

Reply:
Should be good to merge